locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -622,6 +624,14 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
             if (node == null)
                 throw new IllegalStateException("No nodes returned by jclouds create-nodes in " + setup.getDescription());
 
+            boolean windows = node.getOperatingSystem().getFamily().equals(OsFamily.WINDOWS);
+
+            if (windows) {
+                // FIXME: Tidy this up and allow for user-specified credentials
```

Comment:
I'd prefer FIXME/TODO comments to be more specific than "Tidy this up". The second part of comment is very clear. If there is more tidying than that, then please say what specifically you're thinking of.